### DISCUSSION TITLE
テンプレート機能がほしい

### DISCUSSION THREAD

**Discussion by duri0214** [increments/qiita-discussions#142] [要望・アイデア・不具合]
githubのIssueテンプレートのようなマークダウンのテンプレート機能がほしい
```
## はじめに
```

**Comment by kskwtnk** [comment]
@duri0214 

ご意見ありがとうございます！

こちらはGitHub Discussionsにテンプレートのような機能がないため提供できずにいます 😭 
Issueは立っていたのでどこかで提供されるかもしれず、そのときには準備いたします！

**Reply by duri0214** [reply]
あ、ちなみにqiitaの記事編集での話です（qiitaの画面から書いたのでつい「なにを」の表現が抜けてしまいました）

**Reply by kskwtnk** [reply]
@duri0214 
すみません完全に勘違いしていました！ 🙇 

現状、運営内だと議論にはあがっていない項目なのですが、どういったモチベーションで欲しいか伺っても良いでしょうか？
例えば「記事の構成が毎回ほぼ一緒なので見出しだけテンプレート化しておきたい」など、具体的なシーンや用途が分かると非常にありがたいです！

**Reply by duri0214** [reply]
1. はじめに、とか、終わりに、環境情報、など文章構成が決まっている場合とか起承転結のパターンが決まっているような場合。技術の書物だとよくあるはずです。

2. チームで書いてるときに漏れをなくす
まぁgithubと同じです。書いてほしいことを漏らさない「仕組み」が欲しいということですね。この項目埋めてけばとりあえずええんやな？ってなる

**Reply by shiracamus** [reply]
質問投稿のときはデフォルトのテンプレートが展開されますね。
記事投稿の場合、技術記事以外もあるので、テンプレートを数種類用意して選べると良さそうですね。

**Reply by kskwtnk** [reply]
@duri0214 @shiracamus 
ありがとうございます！

まずは検討から開始します！
